src/hotspot/share/classfile/classLoaderExt.cpp

```diff
@@ -260,47 +260,33 @@ InstanceKlass* ClassLoaderExt::load_class(Symbol* name, const char* path, TRAPS)
   assert(DumpSharedSpaces, "this function is only used with -Xshare:dump");
   ResourceMark rm(THREAD);
   const char* class_name = name->as_C_string();
-
   const char* file_name = file_name_for_class_name(class_name,
                                                    name->utf8_length());
   assert(file_name != NULL, "invariant");
 
   // Lookup stream for parsing .class file
   ClassFileStream* stream = NULL;
-  ClassPathEntry* e = find_classpath_entry_from_cache(path, CHECK_NULL);
+  ClassPathEntry* e = find_classpath_entry_from_cache(THREAD, path);
   if (e == NULL) {
-    return NULL;
-  }
-  {
-    PerfClassTraceTime vmtimer(perf_sys_class_lookup_time(),
-                               THREAD->as_Java_thread()->get_thread_stat()->perf_timers_addr(),
-                               PerfClassTraceTime::CLASS_LOAD);
```

Comment:
Why was the above removed?

Reply:
Thanks for the review. It was removed by mistake and I restore it.